src/node_crypto.cc

```diff
@@ -310,7 +310,10 @@ void SecureContext::Initialize(Environment* env, Local<Object> target) {
   env->SetProtoMethod(t, "addCRL", SecureContext::AddCRL);
   env->SetProtoMethod(t, "addRootCerts", SecureContext::AddRootCerts);
   env->SetProtoMethod(t, "setCiphers", SecureContext::SetCiphers);
+  env->SetProtoMethod(t, "hasEC", SecureContext::HasEC);
+#if !defined(OPENSSL_NO_ECDH)
```

Comment:
`openssl_no_ec` or `openssl_no_elliptic_curve` would fit better with the existing `openssl_fips` and `openssl_no_asm`.